client/js/lounge.js

```diff
@@ -404,6 +405,11 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
+		if (type === "settings") {
+			$("#session-list").text("Loading…");
+			socket.emit("sessions:get");
```

Comment:
Are you planning for live updates? A bit gross but at the very least we could setInterval when settings are open? Meh

Reply:
If you want live updating, then you would emit the event from the server whenever anything related to sessions changes. But I feel like would complicate it more than necessary (for now at least).

Reply:
Yeah we'd need `socket.emit("sessions:live");` and `socket.emit("sessions:stop");` when leaving the window or something like that. I get that it's more complicated, but it would also be much more valuable. #986 supported that (though not optimal there, I know) :/
I understand if you don't want to bother with this for this PR.

Reply:
I don't see how it's absolutely necessary to have live refreshing here, this can be a future enhancement.

Reply:
It's not, can totally be a future enhanceement.